src/main/java/teammates/common/util/HibernateUtil.java

```diff
@@ -238,6 +238,16 @@ public static <T extends BaseEntity> T get(Class<T> entityType, Object id) {
         return HibernateUtil.getCurrentSession().get(entityType, id);
     }
 
+    /**
+     * Return an instance, whose state may be lazily fetched.
+     * If there is no such persistent instance, EntityNotFoundException is thrown when the instance state
+     * is first accessed. This method can be used to increase performance, if it is known that the instance exists
+     * and fetching of an entity is not necessary.
+     */
```

Comment:
Great job on this, could you mention that only the ID field in the entity is loaded too?